feature/gnmi/ate_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -130,12 +134,14 @@ func TestInterfaceCounters(t *testing.T) {
 		counter: ipv6Counters.OutPkts().Lookup(t),
 	}, {
 		// desc: "IPv6InDiscardedPkts",
-		path:    ipv6CounterPath + "in-discarded-pkts",
-		counter: ipv6Counters.InDiscardedPkts().Lookup(t),
+		path: ipv6CounterPath + "in-discarded-pkts",
+		// TODO: Uncomment counter in-discarded-pkts after the issue fixed.
+		// counter: ipv6Counters.InDiscardedPkts().Lookup(t),
 	}, {
 		// desc: "IPv6OutDiscardedPkts",
-		path:    ipv6CounterPath + "out-discarded-pkts",
-		counter: ipv6Counters.OutDiscardedPkts().Lookup(t),
+		path: ipv6CounterPath + "out-discarded-pkts",
```

Comment:
Instead of skipping based on `desc` and `counter`, we can skip based on a deviation flag. 

```
tc := {
		desc: "IPv6OutDiscardedPkts",
		path:    ipv6CounterPath + "out-discarded-pkts",
		counter: ipv6Counters.OutDiscardedPkts().Lookup,
}

if (*deviations.SubInterfacePacketCountersSupported) {
 if !tc.counter(t).IsPresent() {
    t.Errorf("Get IsPresent status for path %q: got false, want true", tc.path)
 }
}
```